ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -129,6 +130,13 @@
   // End of copied values.
   );
 
+  private static final Set<String> ALLOWED_SEARCH_CHANNELS = ImmutableSet.of("ts");
+
+  private static final Set<String> MOBILE_SEARCH_AFFECTED_NAMESPACES = ImmutableSet.of(
```

Comment:
I think this could be confusing if there end up being other mobile search-related issues, so I'd suggest either naming this something like `BUG_1751955_AFFECTED_NAMESPACES` or creating a `bool bug1751955Affected()` method.